css/port.css

```diff
@@ -0,0 +1,67 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;1,100;1,200;1,300;1,500;1,600;1,700&display=swap');
+
+html {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+}
+
+.container {
+  max-width: 375px;
+  height: 100vh;
+  margin: auto;
+  background-image: url('../images/header-shapes\ mobile.png');
+}
+
+.header {
+  padding: 0 35px;
+}
+
+.logo {
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 20px;
+  text-decoration: none;
+}
+
+.navbar {
+  width: 100%;
+  background: #fff;
+  display: flex;
+  justify-content: space-between;
+}
```

Comment:
- [ ] Kindly make sure that the content of the navbar are aligned properly, and also add the right margin. I will suggest you add the property `align-items: center;` to it.

Your design
![Capture](https://user-images.githubusercontent.com/85095702/164809029-6b4e599d-efc7-46e2-afba-97ca90e490b2.PNG)

- [ ] Also the background image is too close to the top

Reply:
Thanks alot